diff --git cxx/Makefile.Rules.in cxx/Makefile.Rules.in
index 3c6b6fd..1a0e02c 100644
--- cxx/Makefile.Rules.in
+++ cxx/Makefile.Rules.in
@@ -507,7 +507,7 @@ BOOST_INC= @BOOST_FLAGS@
 endif
 # generically supported c++ options
 CXXOPTS = -g $(ENV_CXXFLAGS) $(PACKAGE_OPTS) $(CXX_TEMPLATES) -D_FREEFREE -D_EATGNU 
-CXXOPTS = -g $(PACKAGE_OPTS) $(CXX_TEMPLATES) -D$(OS_ARCH) -DCCAFE_SPEW=$(CCAFE_SPEW) $(BOOST_INC) $(NOT_HAVE_CXX_LIMITS) $(CCAFE_THREAD_DEFS) $(CCAFE_SOCKET_DEFS) $(CCAFE_CMDSHELL_DEFS)
+CXXOPTS = -g $(ENV_CXXFLAGS) $(PACKAGE_OPTS) $(CXX_TEMPLATES) -D$(OS_ARCH) -DCCAFE_SPEW=$(CCAFE_SPEW) $(BOOST_INC) $(NOT_HAVE_CXX_LIMITS) $(CCAFE_THREAD_DEFS) $(CCAFE_SOCKET_DEFS) $(CCAFE_CMDSHELL_DEFS)
 
 # gnu options (-fall-virtual is broken)
 GPPOPTS = -Wall -fall-virtual -Woverloaded-virtual -D_ODEBUG
